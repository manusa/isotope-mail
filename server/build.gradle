buildscript {
	ext {
		springBootVersion = '2.0.4.RELEASE'
	}
	repositories {
		mavenCentral()
	}
	dependencies {
		classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
	}
}

apply plugin: 'java'
apply plugin: 'eclipse'
apply plugin: 'org.springframework.boot'
apply plugin: 'io.spring.dependency-management'

group = 'com.marcnuri.isotope'
version = '0.0.1-SNAPSHOT'
sourceCompatibility = 1.8

repositories {
	mavenCentral()
}


dependencies {
	compile('org.springframework.boot:spring-boot-starter')
	compile('org.springframework.boot:spring-boot-starter-web')
	compile('org.springframework.boot:spring-boot-starter-hateoas')
	compile('com.sun.mail:javax.mail:1.6.1')
	testCompile('org.springframework.boot:spring-boot-starter-test')
	testCompile('com.marcnuri:spring-common-test:0.0.2')
}
