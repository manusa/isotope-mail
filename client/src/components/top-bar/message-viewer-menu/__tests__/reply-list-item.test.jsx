import React from 'react';
import {shallow} from 'enzyme/build';
import ReplyListItem from '../reply-list-item';

describe('ReplyListItem component test suite', () => {
  let t;
  beforeAll(() => {
    t = jest.fn(arg => arg);
  });
  describe('Snapshot render', () => {
    test('Snapshot render, should render DownloadListItem', () => {
      // Given
      const props = {t, replyAction: jest.fn()};
      // When
      const replyListItem = shallow(<ReplyListItem {...props}/>);
      // Then
      expect(replyListItem).toMatchSnapshot();
    });
  });
});
