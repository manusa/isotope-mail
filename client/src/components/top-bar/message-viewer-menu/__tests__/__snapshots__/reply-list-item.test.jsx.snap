// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReplyListItem component test suite Snapshot render Snapshot render, should render DownloadListItem 1`] = `
<li
  className="mdc-list-item"
  onClick={[MockFunction]}
>
  <span
    className="mdc-list-item__graphic material-icons"
  >
    reply
  </span>
  topBar.messageViewerMenu.reply
</li>
`;
